olca-proto-io/src/main/java/org/openlca/proto/io/output/ProcessWriter.java

```diff
@@ -61,6 +55,50 @@ public ProtoProcess write(Process process) {
     return proto.build();
   }
 
+	private ProtoProcessDocumentation.Builder mapDocOf(ProcessDocumentation d) {
+		var proto = ProtoProcessDocumentation.newBuilder();
+		if (d == null)
+			return proto;
+
+		proto.setTimeDescription(d.time);
+		proto.setTechnologyDescription(d.technology);
+		proto.setDataCollectionDescription(d.dataCollectionPeriod);
+		proto.setCompletenessDescription(d.completeness);
+		proto.setDataSelectionDescription(d.dataSelection);
+		proto.setReviewDetails(d.reviewDetails);
+		proto.setDataTreatmentDescription(d.dataTreatment);
+		proto.setInventoryMethodDescription(d.inventoryMethod);
+		proto.setModelingConstantsDescription(d.modelingConstants);
+		proto.setSamplingDescription(d.sampling);
+		proto.setRestrictionsDescription(d.restrictions);
+		proto.setIsCopyrightProtected(d.copyright);
+		proto.setIntendedApplication(d.intendedApplication);
+		proto.setProjectDescription(d.project);
+		proto.setGeographyDescription(d.geography);
+		proto.setCreationDate(date(d.creationDate));
+		proto.setValidFrom(date(d.validFrom));
+		proto.setValidUntil(date(d.validUntil));
+
+		proto.setReviewer(Refs.refOf(d.reviewer));
+		proto.setDataDocumentor(Refs.refOf(d.dataDocumentor));
+		proto.setDataGenerator(Refs.refOf(d.dataGenerator));
+		proto.setDataSetOwner(Refs.refOf(d.dataSetOwner));
+		proto.setPublication(Refs.refOf(d.publication));
+		d.sources.stream()
+			.map(Refs::refOf)
+			.forEach(proto::addSources);
+
+		return proto;
+	}
+
+	private static String date(Date date) {
```

Comment:
shouldn't we use the same method as in the Json export here to make sure that we always write the same date/time format?

Reply:
OK, my thought was that I should avoid a dependency with the `jsonld` package.